papaparse.js

```diff
@@ -946,10 +946,13 @@
 				_delimiterError = false;
 			}
 
-			if (_config.skipEmptyLines)
+			if (_config.skipEmptyLines || _config.skipNullLines)
 			{
 				for (var i = 0; i < _results.data.length; i++)
-					if (_results.data[i].length === 1 && _results.data[i][0] === '')
+					if (
+						(_config.skipEmptyLines && isEmptyLine(_results.data[i])) ||
```

Comment:
Why creating two functions? It won't be simpler to create a function skipLine which makes the computation of both cases? I don't think each case is to complex to have it's own function.

Reply:
I'll change this. I was following some advice that functions are free, so make a bunch of small, dumb ones. But I can see your point that they can sometimes get _too_ dumb.

Are you thinking the signature would look like this?

`skipLines(_results.data[i], _config.skipEmptyLines, _config.skipNullLines)`?